tests/test_basic.py

```diff
@@ -1367,6 +1367,23 @@ def test_ohmic_inputresistance():
         stimulus_current)
 
 
+def test_ohmic_inputresistance_zero_stimulus_current():
```

Comment:
missing an underscore between input and resistance ;)

Reply:
Fixed, thanks